### PR TITLE
Docs: Update current members' list

### PR DESCRIPTION
**Pull request by anubh-v**
```
Let's update the list of current members to include
the new contributors.
```

### CODE REVIEW

**Comment by anubh-v** [comment]
@reposense/stage0-reviewers @yash-chowdhary @ang-zeyu @niqiukun please take a look if the list is ok

File: docs/Team.md [hunk 1]
```diff
@@ -61,6 +61,28 @@ Contributor [2018 Dec - 2019 May]<br/>
 ![](https://avatars3.githubusercontent.com/u/35756209?s=150&v=4)<br/>
 **Role**: Contributor [2019 May - ]<br/>
 
+### [Liu Yiwen](https://github.com/0blivious)
```
Comment (jamessspanggg) [inline docs/Team.md:64]: Might be better to sort based on start date?

**Comment by jamessspanggg** [comment]
I think can move the contributor jin minjia to a past member. She was working on the project from May 2019 to Aug 2019.

**Comment by anubh-v** [comment]
> I think can move the contributor jin minjia to a past member. She was working on the project from May 2019 to Aug 2019.

Thanks for the response @jamessspanggg 

I will make this change, as well as sort by start date.

File: docs/Team.md (hunk 1 above)
Comment (anubh-v) [reply docs/Team.md:61]: Done

Comment (eugenepeh) [reply docs/Team.md:61]: Yes, i think sorting by start date is better

**Comment by eugenepeh** [comment]
jun an and shupeng have also withdrawn from RepoSense, can move them to the past members.
The end date can be based on their last month of activity.

**Comment by fzdy1914** [comment]
@eugenepeh We should also remove some of the inactive developers from stage1, stage2 reviewers so they do not get spammed by our activity.

**Comment by eugenepeh** [comment]
>@eugenepeh We should also remove some of the inactive developers from stage1, stage2 reviewers so they do not get spammed by our activity.

Have removed the inactive developers. They should be able to leave the team too on their own if they felt that theyre spammed i think

**Comment by anubh-v** [comment]
> jun an and shupeng have also withdrawn from RepoSense, can move them to the past members.

Done

**Pull request merged**